app.json

```diff
@@ -1,6 +1,6 @@
 {
     "runtime": {
-        "arguments": "",
+        "arguments": "--v=1",
```

Comment:
Are these changes we want in dev/master? Or should this be in local.json?

Reply:
correct, will revert.